Makefile

```diff
@@ -46,16 +49,12 @@ test-coverage:
 check:
 	poetry run black --check --diff .
 	poetry run isort --check --diff .
-	poetry run flake8 . --exclude ${FLAKE8_EXCLUDE}
+	poetry run flake8 . --exclude ${FLAKE8_EXCLUDE} --ignore=E501,E201,E202,W503
```

Comment:
the fact that we need to exclude these makes me think we don't have the linting setup right
see also https://github.com/vliz-be-opsci/py-sync-fs-triples/issues/8

not something to try and fix now though

Reply:
issue mentioned in #14